jsonschema/catalog.json

```diff
@@ -21,12 +12,15 @@
       ]
     },
     {
-      "url": "https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/cloudbuild/v1/BuildEventData.json",
-      "name": "BuildEventData",
-      "description": "Build event data\n Common build format for Google Cloud Platform API operations.\n Copied from\n https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudbuild/v1/cloudbuild.proto.",
-      "datatype": "google.events.cloud.cloudbuild.v1.BuildEventData",
+      "url": "https://googleapis.github.io/google-cloudevents/jsonschema/google/events/cloud/firestore/v1/DocumentEventData.json",
```

Comment:
Why are all these changes here? Is the ordering non-deterministic at the moment, or was it non-deterministic before and this just corrects it?

Reply:
It's currently non-deterministic. I state that the PR will be smaller due to this issue in the PR description.

I need to merge https://github.com/googleapis/google-cloudevents/pull/115 before this PR to not see this artifact.

Reply:
Ah, right. I'd missed that.